Documentation/Developer/Index.rst

```diff
@@ -0,0 +1,38 @@
+.. include:: ../Includes.txt
+
+
+.. _developer:
+
+================
+Developer Corner
+================
+
+Build your own widget
+=====================
+
+.. todo::
+
+   define how developers can build their own widgets
+
+-------------------------
+Extend predefined widgets
+-------------------------
+
+The extension ships a list of predefined widgets to ensure a consistent look of tables, graphs and data.
+
+The shipped widgets are in the folder `Classes/Widgets`.
+
+
+-------------------------------
+Make a completely custom widget
```

Comment:
To be honest, we don't want to promote this. We want people to use the predefined widget types as much as possible to make sure we keep an unified dashboard.

Reply:
so remove it